desktop-app/app/components/UserPreferences/index.js

```diff
@@ -245,6 +247,29 @@ function UserPreference({
           </p>
         </div>
       </div>
+      <div className={commonClasses.sidebarContentSectionContainer}>
+        <div
+          className={cx(
+            commonClasses.flexAlignVerticalMiddle,
+            classes.sectionHeader
+          )}
+        >
+          Start Page
+        </div>
+        <div className={classes.marginTop}>
+          <Select
+            options={startOptions}
+            value={
+              startOptions.find(x => x.value === userPreferences?.startPage) ||
+              startOptions[0]
+            }
+            onChange={val => {
+              notifyStartPreferenceChanged(val.value);
```

Comment:
@bikingbadger Where is this notification consumed?

Reply:
Currently, it isn't and is something that probably needs discussion. When changed should it update the current page? Or only on the next start? This part could then be removed if it is only done on startup.

Reply:
I think I shouldn't refresh the current page, it would take effect on the next startup